tsc/meetings/2020-01-08.md

```diff
@@ -0,0 +1,71 @@
+         * Finalizing switching our testing pipeline fully over to always run in both Python 2 and
+           3 environments.
+         * Maintaining Python 2 support for the foreseeable future. It may take a while for users
+           to switch over. Let's reevaluate the pace of the Python 3 transition in a few months.
+   * Greg
+      * Working on the plugin / messaging system.
+      * Minor bug fixes.
+      * Spring boot update is fully released now.
+   * Christian
+      * Merged in Brian's Python 3 changes, seems to be working well.
+      * Working on RQD windows support.
+      * No major blockers yet.
+      * RQD attempts to assume UIDs/GIDs. This isn't supported on Windows, we need to figure out
+        another way to handle that.
+      * Users/permissions system is related to this.
+        * This is a big security overhaul and is complex but important to some users. We should
+          consider it high priority on the roadmap.
+        * When implementing this will need to consider users' existing user systems e.g.
+          ActiveDirectory, LDAP. Also a lot of work.
+        * Let's not block Windows support on this. Windows support is much higher priority.
+        * It's fine to have a configurable user that RQD will assume for all jobs. Probably
+          separate config for Linux and Windows users. This can include not only the RQD process's
+          user but the user under which job commands are actually executed. The latter currently
+          defaults to the same username that submitted the job. We will need to assume that that
+          default user has access to all necessary files.
+      * Client tools on Windows: still working on it. Getting close but working on it in parallel
+        with RQD so as to fully test the job workflow. So the UID issue in RQD is blocking this
+        for now. Should be resolved soon.
+   * Lars
+      * Working on removing SPI-specific code. Filed a few issues
+         * Issues include how Arnold jobs use their cores, calculate_core_hours, cudainfo, etc.
+      * Finding Nimby problems when using Teradici.
+         * Ben: SPI uses Teradici without an issue. Will follow up to see what SPI did here to get
+           it working, if anything.
+      * Checkpointing
+         * RQD can update frames to a "checkpointing" state in the database.
+         * Discussion of various checkpointing models and what we might need to do to support those.
+         * Whatever the solution, we want to leave the door open to alternate models of
+           checkpointing. May even include machine level checkpointing but this hasn't been seen
+           much.
+      * PR in review to change mcp -> scratch
+         * Discussion of scratch vs tmp. Scratch is semi-persistent storage that needs to be
+           manually cleaned up by some external tool. Tmp is generic temp storage that is managed
+           by the OS.
+         * mcp is meant to be tmp storage rather than scratch.
```

Comment:
@larsbijl Could you verify the details of this conversation? Did we reach some conclusion here? I.e. are we planning to keep the PR as-is, switch it to be named more like tmp storage, something else?

Reply:
I think we can move forward as is. if scratch is local or not to a host is irrelevant to opencue, The scheduler doesn't use it to make during placement.

Reply:
Sounds good to me. Added to notes.